src/transport/SessionManager.cpp

```diff
@@ -195,25 +195,44 @@ CHIP_ERROR SessionManager::SendPreparedMessage(SessionHandle sessionHandle, cons
 
     if (sessionHandle.IsSecure())
     {
-        // Find an active connection to the specified peer node
-        SecureSession * session = GetSecureSession(sessionHandle);
-        if (session == nullptr)
+        if (sessionHandle.IsGroupSession())
         {
-            ChipLogError(Inet, "Secure transport could not find a valid PeerConnection");
-            return CHIP_ERROR_NOT_CONNECTED;
+            chip::Transport::PeerAddress multicastAddress =
+                Transport::PeerAddress::Multicast(sessionHandle.GetFabricIndex(), sessionHandle.GetGroupId().Value());
+            destination = static_cast<Transport::PeerAddress *>(&multicastAddress);
+            char addressStr[72];
```

Comment:
There is a constant for this: `Inet::IPAddress::kMaxStringLength`.

Reply:
`Inet::PeerAddress::kMaxToStringSize` since it's a `PeerAddress` not an `IPAddress`.

(And I just noticed that that constant isn't built from `IPAddress::kMaxStringLength` even though it uses `IPAddress::ToString()` — I'll fix that.)